hist/histv7/inc/ROOT/RHistImpl.hxx

```diff
@@ -386,7 +621,11 @@ public:
 
    /// Retrieve the fill function for this histogram implementation, to prevent
    /// the virtual function call for high-frequency fills.
-   FillFunc_t GetFillFunc() const final { return (FillFunc_t)&RHistImpl::Fill; }
+   FillFunc_t GetFillFunc() const final { 
+      auto testFill = (FillFunc_t)&RHistImpl::Fill;
+      testFill = nullptr;
+      return (FillFunc_t)&RHistImpl::Fill; 
+   }
 
    /// Apply a function (lambda) to all bins of the histogram. The function takes
    /// the bin reference.
```

Comment:
What should we do about the Apply function family in the new design? I can personally think of three options:

1. Make it iterate over all bins, not just regular bins but also overflow bins.
2. Duplicate the three `Apply` functions with versions for overflow bins.
3. Give up on the idea of having `Apply` functions, which aren't used anywhere at the moment. After all, with the new non-type-erased RHist, "regular" histogram iteration should not incur one vtable overhead per bin, which was the original motivation for having those functions.

Personally, option 3 has my vote, unless @Axel-Naumann has a strong argument against it.